File: Python/Ad-Hoc/1030.py
nc = int(input())
cases = 0

for _ in range(nc):
    qnt, salto = map(int, input().split())

    sobrevivente = 0
    for i in range(2, qnt + 1):
        sobrevivente = (sobrevivente + salto) % i

    cases += 1
    print(f"Case {cases}: {sobrevivente + 1}")

File: Bee/2_Ad-Hoc/1030.c
#include <stdio.h>

int main()
{
    int nc = 0, i = 0, qnt = 0, salto = 0, j = 0,cases = 0;
    scanf("%d", &nc);
    for (i = 0; i < nc; i++)
    {
        scanf("%d %d", &qnt, &salto);
        int sobrevivente = 0;
        for (j = 2; j < qnt + 1; j++)
        {
            sobrevivente = (sobrevivente + salto) % j;
        }
        cases++;
        printf("Case %d: %d\n",cases,sobrevivente+1);
    }

    return 0;
}